### DISCUSSION TITLE
get list of existing endpoints

### DISCUSSION THREAD

**Discussion by softwarebloat** [fastapi/fastapi#2552] [Questions] [answered]
Hi all,
so,it's not really related to fastapi but, i'm looking for a way to check if an endpoint exists without really execute it.
for example, if i have a POST endpoint like `/my/endpoint/path`, how can i just check that this path is exposed/implemented?

And... Does this makes sense?
I need it because i have a boilerplate project used to build a service but i need to know that the user that implements the service doesn't delete the endpoint required.

Feel free to delete the discussion if you think is too OT :)

**Comment by Mause** [comment, marked as answer]
Why not simply check the openapi.json endpoint?

**Reply by softwarebloat** [reply]
damn, you are right. didn't think about it 😅 
thx!!
